### PR TITLE
Update and rename projects.md to open-source-products.md

### PR DESCRIPTION
**Pull request by tacurran**
Changed numerous text portions to better explain the Ory Ecosystem. This needs further attention along with FAQ

### CODE REVIEW

File: docs/docs/ecosystem/open-source-products.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+id: products
```
Comment (aeneasr) [inline docs/docs/ecosystem/open-source-products.md:2]: Regarding IDs and file names, see my comment in the base review comment. This must match the file name, so if this was `projects.md` it needs to be:

```suggestion
id: projects
```

File: docs/docs/ecosystem/open-source-products.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Ory is an open source ecosystem of services with clear functional boundaries that solve
+authentication and authorization:
+
+- Ory Kratos is an identity management server;
+- Ory Hydra is an OAuth 2.0 and OpenID Connect provider;
+- Ory Oathkeeper is an Identity and Access Rule-based Proxy; and
+- Ory Keto is an access control system.
+
+Each service works standalone and has an versioned API. Combine these Open Source products for the full
```
Comment (aeneasr) [inline docs/docs/ecosystem/open-source-products.md:16]: We do not yet have versioned APIs, so how about:

```suggestion
Each service works standalone and has robust REST and sometimes gRPC APIs. Combine these Open Source products for the full
```

File: docs/docs/ecosystem/open-source-products.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+users are allowed to write blog posts. While this is the basis for most
+applications, access control becomes complex as an
+application grows. What started out with a user's username and password grows to
+for example, to machine-2-machine interaction, third party developers accessing your
+user's data, and micro service system architecture.
+
+Ory's projects solve the basic use case and scale to more complex scenarios without painful and slow upgrade
+processes.
+
+![Ory Kratos](https://raw.githubusercontent.com/ory/meta/master/static/logos/logo-kratos.svg)
```
Comment (aeneasr) [inline docs/docs/ecosystem/open-source-products.md:30]: Make it a headline:

```suggestion
## ![Ory Kratos](https://raw.githubusercontent.com/ory/meta/master/static/logos/logo-kratos.svg)
```

File: docs/docs/ecosystem/open-source-products.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+rewrite every aspect of identity management for every project. Ory Kratos implements all
+common flows such as login and logout, account activation, mfa/2fa, profile and
+session management, user facing errors and account recovery methods. For example, 
+an Ory Docker Image provides the basis to write a simple UI for numerous languages or frameworks
+in compliance with GDPR, common conventions such as address verification data protection and privacy security. 
+Ory Kratos applies security standards established by experts including National Institute of
+Sciences NIST, Internet Engineering Task Force IETF, Microsoft Research, Google
+Research, and Troy Hunt. It is also possible to customise user flows for flexible usage.
+
+![Ory Hydra](https://raw.githubusercontent.com/ory/meta/master/static/logos/logo-hydra.svg)
```
Comment (aeneasr) [inline docs/docs/ecosystem/open-source-products.md:43]:
```suggestion
## ![Ory Hydra](https://raw.githubusercontent.com/ory/meta/master/static/logos/logo-hydra.svg)
```

File: docs/docs/ecosystem/open-source-products.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+Ory Hydra enables an application to be an OAuth 2.0 and OpenID Connect provider. Hydra
+supportds applications that work on different
+devices, that have machine-2-machine interaction, or enable third-party
+developers to use an API (and pay for it). Ory Hydra is not identity management, though. Hydra connects to 
+ identity management systems, e.g. the one from the paragraph above, or any
+MySQL+PHP login service, or a Federated SAML SSO and is capable of issuing,
+in a secure and OpenID Certified manner, access, refresh, and ID tokens. Hydra can be used in a 5MB Docker Image with almost no configuration
+required.
+
+![Ory Oathkeeper](https://raw.githubusercontent.com/ory/meta/master/static/logos/logo-oathkeeper.svg)
```
Comment (aeneasr) [inline docs/docs/ecosystem/open-source-products.md:54]:
```suggestion
## ![Ory Oathkeeper](https://raw.githubusercontent.com/ory/meta/master/static/logos/logo-oathkeeper.svg)
```

File: docs/docs/ecosystem/open-source-products.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+Ory Oathkeeper prevents unqualified user access to an application through advanced permission checks and access controls. Ory Oathkeeper defines access rules
+for API endpoints e.g. OAuth 2.0 Access Token certain set of
+permissions, a valid JSON Web Token, a valid SAML assertion addiding a firewall like service 
+in front of cloud services.
+
+![Ory Keto](https://raw.githubusercontent.com/ory/meta/master/static/logos/logo-keto.svg)
```
Comment (aeneasr) [inline docs/docs/ecosystem/open-source-products.md:61]:
```suggestion
## ![Ory Keto](https://raw.githubusercontent.com/ory/meta/master/static/logos/logo-keto.svg)
```

**Review by aeneasr — CHANGES_REQUESTED** [review]
Great stuff!

One important thing is to keep file names the same or if they change, ensure the following three things:

1. Update the side bar to link to the new document and delete the old one: https://github.com/ory/docs/blob/master/docs/sidebar.json
2. Update all references to the file you deleted / renamed
3. Add a 302 redirect to the website so Google search results, SEO, and inbound links don't break. Example: https://github.com/ory/web/blob/e3835c77e93786efb3bfd2ee416e7eb36f73a16a/static/_redirects#L257

My suggestion is to keep it the same for this PR!

**Comment by tacurran** [comment]
@vinckr can you please implement the changes required from @aeneasr ?  I will not get to it this week.

**Comment by vinckr** [comment]
Hey @aeneasr, I addressed your feedback in this PR: 
https://github.com/ory/docs/pull/515.
I just changed the document title and left the filename the same, since it only shows up in the URL should be fine, right? 
And thus we don't need more renaming, redirects etc.

**Review by vinckr — APPROVED** [review]

**Comment by aeneasr** [comment]
Merged via #515

**Pull request closed without merging**